### ISSUE TITLE
Protocol https not supported or disabled in libcurl

### ISSUE THREAD

**Issue by tn430** [alisw/alibuild#485]
[log.txt](https://github.com/alisw/alibuild/files/1302198/log.txt)
Hello,

I am currently trying to build AliPhysics with aliBuild. Everything worked fine until this:

DEBUG:AliPhysics:cgal:0: curl: (1) Protocol https not supported or disabled in libcurl
DEBUG:AliPhysics:cgal:0: 
ERROR:AliPhysics:cgal:0: Error while executing /home/alissa/alice/sw/SPECS/ubuntu1604_x86-64/cgal/v4.6.3-1/build.sh on `alissa-XXX'.
ERROR:AliPhysics:cgal:0: Log can be found in /home/alissa/alice/sw/BUILD/cgal-latest/log.
ERROR:AliPhysics:cgal:0: Please attach this file if you intend to request support.
ERROR:AliPhysics:cgal:0: Build directory is /home/alissa/alice/sw/BUILD/cgal-latest/cgal.

curl doesn't seem to support https. I tried to fix this by configuring curl with '--with-ssl', but when I type 'curl -V' I still get:

curl 7.18.2 (i686-unknown-linux-gnu) libcurl/7.18.2
Protocols: tftp ftp telnet dict http file 
Features: IPv6 Largefile 
So no sign of https.

Did someone have the same issue? I would be very grateful if I get some help.

My log file is attached.

**Comment by ktf** [comment]
IIRC we use system curl, so that's a problem with your installation, not with alibuild itself. Can you confirm this by doing:

```
curl -Lo cgal.tar.bz2 https://gforge.inria.fr/frs/download.php/file/35136/ 
```

in a shell?

**Comment by tn430** [comment]
Thank you for your fast answer. You are right. I tried and I still get the same error. Do you have an idea how I can fix this?

**Comment by ktf** [comment]
Do you still have this problem? Which distribution are you using?

**Comment by tn430** [comment]
No I solved it.

I used 
> curl -V
It showed me that even though I reinstalled curl manually with the command "--with-ssl", https was still not supported.
Then I used
> which curl
This showed me that the version being used was not at all the one I installed manually, it was one installed by urtool, a (a cross compiler). So I just went to the directory given by the command and deleted curl there.

So nothing wrong with alibuild!
